### DISCUSSION TITLE
4주차 미팅

### DISCUSSION THREAD

**Discussion by nyj001012** [nyj001012/ft_irc#30] [General]
# 논의할 내용
## 제출일 가시화

**Comment by nyj001012** [comment]
## 진행상황
- #27
- #25

**Comment by nyj001012** [comment]
## 분담
- IRC 기능
- irssi를 클라이언트로 하여 동작 확인

## 제출일 가시화
1차: 다음 주 평일
2차: 그 다음 주

**Comment by sejoonkimmm** [comment]
@bartshin 님이 작성해주신 requesthandler의 Task 클래스에서 IRC 기능들을 덧붙이고 있습니다!
문제는 Mac 환경에서는 std::auto_ptr을 인식하지 못하는 건데... create 함수 만들어진 것에서 수정을 덧붙이고 있긴 한데 사용해도 되는 건지 알아보아야 할 것 같습니다

**Reply by sejoonkimmm** [reply]
+ 명령어 사용 성공했을 때 클라이언트에게 알려주거나 (예 : 유저 생성 성공 시 유저 생성이 되었다) 하는 로직이 어디에 있는지 알려주실 수 있을까요? 제가 IRC 기능 만들 때도 그 코드로 합쳐졌으면 좋겠습니다.
